.github/workflows/performance.yml

```diff
@@ -152,7 +160,34 @@ jobs:
         run: npm run test:performance
 
       - name: Print performance tests results
-        run: "node ./tests/performance/results.js"
+        run: node ./tests/performance/results.js
+
+      - name: Check out target commit (target branch or previous commit)
+        run: |
+          if [[ -z "$TARGET_REF" ]]; then
+            git fetch -n origin $TARGET_SHA
+          else
+            git fetch -n origin $TARGET_REF
+          fi
+          git reset --hard $TARGET_SHA
+        env:
+          TARGET_REF: ${{ github.event_name == 'pull_request' && github.event.pull_request.base.ref || '' }}
+          TARGET_SHA: ${{ github.event_name == 'pull_request' && github.event.pull_request.base.sha || github.event.before }}
+
+      - name: Install npm dependencies
+        run: npm ci
+
+      - name: Build WordPress
+        run: npm run build
+
+      - name: Run target performance tests (base/previous commit)
+        run: npm run test:performance -- --prefix=before
+
+      - name: Print target performance tests results
```

Comment:
Would it be helpful to define this as a script in the `package.json` file? Not sure if being able to say `npm run test:performance-results` would be useful. But it would definitely make moving or changing the command easier down the road having to only change it in one place.

Reply:
Probably best to do that in a future step.

Right now we have 3 different scripts:

* `results.js` - print results for an individual run
* `compare-results.js` - compare results from 2 runs and print a result (console + markdown)
* `log-results.js` - send results to dashboard

I envision a future where `results.js` is absorbed in the test runner, i.e. running tests automatically prints nice results at the end.